### PR TITLE
🚀 Improve `/skript info`

### PR DESCRIPTION
**Pull request by AyhamAl-Ali**
### Description

Just a little improvement to an annoying result in `/skript info` when a user doesn't have addons it makes others feel like the user didn't upload the full result message.

**Before**
image

**After**
image

**NEW**
- Added dependencies info section
image

---
**Target Minecraft Versions:** Any
**Requirements:** None
**Related Issues:** None

### CODE REVIEW

File: src/main/java/ch/njol/skript/SkriptCommand.java [hunk 1]
```diff
@@ -324,7 +324,7 @@ public boolean onCommand(@Nullable CommandSender sender, @Nullable Command comma
 				info(sender, "info.documentation");
 				info(sender, "info.server", Bukkit.getVersion());
 				info(sender, "info.version", Skript.getVersion());
-				info(sender, "info.addons");
+				info(sender, "info.addons", Skript.getAddons().isEmpty() ? "None" : "");
```
Comment (TPGamesNL) [inline src/main/java/ch/njol/skript/SkriptCommand.java:327]:
```suggestion
				info(sender, "info.addons", Skript.getAddons().isEmpty() ? "none" : "");
```

**Review by TPGamesNL — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/SkriptCommand.java (hunk 1 above)
Comment (bluelhf) [reply src/main/java/ch/njol/skript/SkriptCommand.java:327]: I'd argue the uppercase "None" is correct in this case:
In both British English and American English, separate sentences after a segmental use of a colon are capitalised. Also, using `None` would be consistent with plugin names, which are usually in pascal case (with some rare exceptions like `spark`)

Comment (AyhamAl-Ali) [reply src/main/java/ch/njol/skript/SkriptCommand.java:327]: I think blue have given the official side of the answer, adding to that it just looks better capitalized also following the plugin naming.

Comment (Moderocky) [reply src/main/java/ch/njol/skript/SkriptCommand.java:327]: Agree with @bluelhf changes aren't needed.

**Comment by Moderocky** [comment]
Is there any other missing information that should be added to the `info` command?

**Comment by AyhamAl-Ali** [comment]
> Is there any other missing information that should be added to the `info` command?

Not sure but this could be added to #3916

**Comment by bluelhf** [comment]
> 
> 
> Is there any other missing information that should be added to the `info` command?

Possibly the versions of any plugins Skript hooks into?

File: src/main/java/ch/njol/skript/SkriptCommand.java (hunk 1 above)
Comment (TPGamesNL) [reply src/main/java/ch/njol/skript/SkriptCommand.java:327]: Okay

File: src/main/resources/lang/english.lang [hunk 2]
```diff
@@ -140,7 +140,8 @@ skript command:
 		documentation: Skript's documentation can be found here: <aqua>https://skriptlang.github.io/Skript
 		version: Skript Version: <aqua>%s
 		server: Server Version: <aqua>%s
-		addons: Installed Skript Addons:
+		addons: Installed Skript Addons: <aqua>%s
+		dependencies: Installed dependencies: <aqua>%s
```
Comment (TPGamesNL) [inline src/main/resources/lang/english.lang:144]: Should be applied to all language files

**Review by TPGamesNL — CHANGES_REQUESTED** [review]

**Review by TPGamesNL — APPROVED** [review]

**Pull request merged**